### ISSUE TITLE
About the graph constraint in the evaluation of MEET.

### ISSUE THREAD

**Issue by ChCh1999** [visinf/veto#5]
Under the graph constraint, the evaluator is supposed to take only one prediction for each subject-object pair.
However, in the implement of MEET, you just concatenates the `pred_rel_scores` and `pred_rel_labels` of each group. That means the evaluator will include 5 prediction for each subject-object pair, which might result in an unfair comparison.
Have you noticed this in your experiments? 
https://github.com/visinf/veto/blob/832ba0e4f79595fad60564a9f2409472b84a15a9/pysgg/modeling/roi_heads/relation_head/inference.py#L376-L397

**Comment by ZHUXUHAN** [comment]
Yes, i also question this problem in (https://github.com/visinf/veto/issues/1), this unfair comparison should be attentioned by the researchers, because some reviewers will ask to compare this paper.
